### ISSUE TITLE
Missing files in .fsproj files are not surfaced as errors

### ISSUE THREAD

**Issue by rookboom** [ionide/ionide-vscode-fsharp#606]
This is really an [FsAutoComplete bug](https://github.com/fsharp/FsAutoComplete/issues/235) but Ionide can do more to warn about missing files.

The following project file will cause FsAutoComplete to crash:

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp2.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <Compile Include="MyModule.fs" />
    <Compile Include="MyMissingModule.fs" />
    <Compile Include="Program.fs" />
  </ItemGroup>

</Project>

```

The symptom in Visual Studio Code is the following message in the Output Window for the F# Language Service:
```

[13:57:56 INFO ] Logging to output at level INFO. If you want detailed messages, try level DEBUG.
[13:58:05 ERROR] RES (003) <- {parse} ERROR in 1018 ms: {stack: Error: connect ECONNREFUSED 127.0.0.1:8842
	at Object.exports._errnoException (util.js:1050:11)
	at exports._exceptionWithHostPort (util.js:1073:20)
	at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1097:14), message: connect ECONNREFUSED 127.0.0.1:8842, code: ECONNREFUSED, errno: ECONNREFUSED, syscall: connect, address: 127.0.0.1, port: 8842, config: [object Object], request: [object Object], response: undefined} Data=undefined

```
There is no warning about the real error (a missing file). It just crashes. At this point error highlighting, autocomplete, jump to reference etc in Ionide will be broken and the only way to fix it is to correct the fsproj file. But how would you know to do this? There is no indication anywhere that there is a missing file. An error in the task bar explaining the problem would be great.

FsAutoComplete could fail more gracefully, but Ionide should really highlight the fact that there are missing references in the fsproj file.

**Comment by Krzysztof-Cieslak** [comment]
CC: @enricosada

**Comment by enricosada** [comment]
@rookboom  thank for the really awesome bug description (on both repo too!) and investigation

@Krzysztof-Cieslak afaik is something recent, was ok some version ago.
if a file not existed, it was shown in project explorer and when clicking that Code asked about `File not exists, want to create it`

**Comment by enricosada** [comment]
fixed in FSAC with https://github.com/fsharp/FsAutoComplete/commit/ba77f71ca8aaaf9e25103444bcdf7eaca9f8d1e2

**Comment by Krzysztof-Cieslak** [comment]
Fixed in `3.8.2`
